bin/proxy

```diff
@@ -141,23 +141,42 @@ function pick_one_virtual_host_from_file() {
     local EXCEPT=$2
 
     local HOST_STRING
-    local HOSTS
+    local HOSTS=()
     local HOST
+    local HOST_OPTIONS=()
 
     HOST_STRING=$(sed -nr 's/.*VIRTUAL_HOST=(.*)/\1/p' "$FILE")
 
     IFS="," read -ra HOSTS <<< "$HOST_STRING"
 
     for HOST_ENTRY in "${HOSTS[@]}"; do
-      if [ "$HOST_ENTRY" != "$EXCEPT" ]; then
-          HOST=$HOST_ENTRY
-          break;
-      fi
+        if [ "$HOST_ENTRY" != "$EXCEPT" ]; then
+            HOST_OPTIONS+=("$HOST_ENTRY")
+        fi
     done
 
-    if [ -z "$HOST" ]; then
-        output_error "No VIRTUAL_HOST found in $FILE."
+    if [ "${#HOST_OPTIONS[@]}" -eq "0" ]; then
+        output_error "No VIRTUAL_HOST overrides found in $FILE."
         exit 1
+    elif [ "${#HOST_OPTIONS[@]}" -gt "1" ]; then
+        PS3="Select a Proxy URL (or quit to exit): "
```

Comment:
Wording threw me off a little. "or type quit to exit" might be a little more clear